mlrun/model_monitoring/model_monitoring_batch.py

```diff
@@ -434,6 +430,40 @@ def _get_drift_status(
         return drift_status
 
 
+def calculate_inputs_statistics(
+    sample_set_statistics: dict, inputs: pd.DataFrame
+) -> dict:
+    """
+    Calculate the inputs data statistics for drift monitoring purpose.
+
+    :param sample_set_statistics: The sample set (stored end point's dataset to reference) statistics. The bins of the
+                                  histograms of each feature will be used to recalculate the histograms of the inputs.
+    :param inputs:                The inputs to calculate their statistics and later on - the drift with respect to the
+                                  sample set.
+
+    :return: The calculated statistics of the inputs data.
```

Comment:
change `return` to `returns`

Reply:
I think we rather stick to the default output of PyCharm's doc string """ enter action. In my case, by default it is completing `:return:` and not `:returns:`. Is it different in your PyCharm?